modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -363,4 +365,22 @@ static CatalogZoneDescriptor zoneOrThrow(Catalog catalog, String name) throws Ca
     public static String pkIndexName(String tableName) {
         return tableName + "_PK";
     }
+
+    /**
+     * Returns index with given name.
+     *
+     * @param schema Schema to look up index in.
+     * @param name Name of the index of interest.
+     * @return Table with given name.
+     * @throws IndexNotFoundValidationException If index with given name is not exists.
+     */
+    static CatalogIndexDescriptor indexOrThrow(CatalogSchemaDescriptor schema, String name) throws IndexNotFoundValidationException {
+        CatalogIndexDescriptor index = schema.index(name);
+
+        if (index == null) {
+            throw new IndexNotFoundValidationException(format("Index with name '{}.{}' not found", schema.name(), name));
```

Comment:
In this class there were methods `schemaOrThrow` and `tableOrThrow`, I don’t think that at the moment there is a need to spoil the consistency in the naming, plus the semantics and documentation of the method indicate that it will be a validation exception that will be thrown.

Reply:
If the whole `CatalogUtils` is about validation stage, the class itself can be renamed. Or these methods might be extracted to a class that's about validation stage logic.

Reply:
I think it's worth fixing this outside of the current task.